dogma_test.go

```diff
@@ -15,22 +15,60 @@
 package centraldogma
 
 import (
-	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
+	"net/url"
 	"testing"
+
+	"github.com/veqryn/h2c"
+	"golang.org/x/net/http2"
 )
 
-// setup sets up a test HTTP server along with a github.Client that is
+const token = "fdbe78b0-1d0c-4978-bbb1-9bc7106dad36"
```

Comment:
nit: How about adding `appToken-` to the front? Because every application token must start with it. 
https://github.com/line/centraldogma/blob/master/server/src/main/java/com/linecorp/centraldogma/server/internal/metadata/Tokens.java#L133

Reply:
Fixed Thank you~!